ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+<!--
+
+IMPORTANT! YOU MUST FOLLOW THESE INSTRUCTIONS OR YOUR ISSUE WILL BE CLOSED.
+
+Thank you for contributing to the Firebase community!
+
+Have a usage question about Functions?
+======================================
+We get lots of those and we love helping you, but the samples issue tracker is not the best place for them and they
+will be closed. Here are some resources to get help:
+
+- Go through the Developer's Guide: https://github.com/angular/angularfire2#developer-guide
+- Read the full API reference: https://github.com/angular/angularfire2/blob/master/docs/api-reference.md
+
+If the official documentation doesn't help, try asking through our officially supported channels:
+
+- Firebase Google Group: https://groups.google.com/forum/#!forum/firebase-talk
+- Stack Overflow: https://stackoverflow.com/questions/tagged/cloud-functions (include the firebase tag, too!)
```

Comment:
link should be https://stackoverflow.com/questions/tagged/google-cloud-functions+firebase can then nuke the `(include the firebase tag, too!)`